packages/manager/src/features/Profile/SSHKeys/CreateSSHKeyDrawer.tsx

```diff
@@ -45,6 +47,17 @@ const CreateSSHKeyDrawer = ({ open, onClose }: Props) => {
 
   const generalError = hasErrorFor('none');
 
+  const SSHTextAreaHelperText = () => (
+    <>
+      <Link to="https://www.linode.com/docs/guides/use-public-key-authentication-with-ssh/">
+        Learn about
+      </Link>{' '}
+      uploading an SSH key or generating a new key pair. Note that the public
+      key begins with <StyledInlineCode>ssh-rsa</StyledInlineCode> and ends with{' '}
+      <StyledInlineCode>your_username@hostname</StyledInlineCode>
```

Comment:
Is there any way we can use the `<Code />` component here? Could we update the existing  `<Code />` component to have your styles? I think yours looks better than what  `<Code />` has now...

It would be nice to use the same component so we can avoid inconsistencies later but maybe this warrants a follow up. 

```suggestion
      <Code>your_username@hostname</Code>
```

Reply:
Great suggestion, wasn't aware of this component. Since it is used only in one other place I went ahead, used it and improved the styles